src/task_types/resmoke_tasks.rs

```diff
@@ -165,6 +165,12 @@ pub struct SubSuite {
     /// List of tests belonging to sub-suite.
     pub test_list: Vec<String>,
 
+    /// Suite the generated suite is based off.
```

Comment:
Here are the new fields on the `SubSuite` struct that the config_writer is using. 

The `origin_suite` is which resmoke suite to base the configuration off and the `exclude_test_list` is used by _misc suites as what tests to exclude.